src/main/java/gtPlusPlus/xmod/gregtech/api/metatileentity/implementations/base/GregtechMeta_MultiBlockBase.java

```diff
@@ -973,6 +979,11 @@ public boolean checkRecipeGeneric(
         // e.g. 100% speed bonus = 200% speed = 100%/200% = 50% recipe duration.
         aSpeedBonusPercent = Math.max(-99, aSpeedBonusPercent);
         float tTimeFactor = 100.0f / (100.0f + aSpeedBonusPercent);
+
+        if (mUseMultiparallelMode) {
```

Comment:
With the placement here you can overclock the batch itself, so you could theoretically 1 tick the batch of 128. Intended?

Reply:
as it stands, yes